src/Wrapper.sol

```diff
@@ -38,12 +38,22 @@ contract Wrapper {
     /* PUBLIC */
 
     /// @dev Compliant to `ERC20Wrapper` contract from OZ for convenience.
-    function depositFor(address account, uint256 amount) public returns (bool) {
+    function depositFor(address account, uint256 value) public returns (bool) {
```

Comment:
Why change to `value` ?
Amount was clear imo

Reply:
to comply with OZ naming